Antidote/ActiveCallViewController.h

```diff
@@ -64,11 +65,26 @@
 @property (assign, nonatomic) BOOL speakerSelected;
 
 /**
+ * Set video button selected.
+ */
+@property (assign, nonatomic) BOOL videoButtonSelected;
+
+/**
  * Set whether or not the resume button is hidden.
  */
 @property (assign, nonatomic) BOOL resumeButtonHidden;
 
 /**
+ * Video view of friend
+ */
+@property (strong, nonatomic) UIView *videoView;
+
+/**
+ * Preview layer of video sending
+ */
+@property (strong, nonatomic) CALayer *previewLayer;
```

Comment:
You can make this properties private and add public methods instead
```
- (void)provideVideoView:(UIView *)view;
- (void)providePreviewLayer:(CALayer *)layer;
```

I think that this would make API more explicit. You actually don't read this properties out of controller.